tests/unit/site.spec.ts

```diff
@@ -38,23 +38,43 @@ describe('# SITE', () => {
     expect(siteB.name).to.be.eq('nameB');
     expect(siteB.location).to.be.eq('marioLand');
     expect(siteB.timezone).to.be.eq('est');
-  }),
-    it('Insert a new site.', async () => {
-      await ChaiRequest('put', `${URL}/`, {
-        name: 'TEST SITE',
-        location: 'TEST',
-        timezone: 'EST',
-      });
+  });
 
-      const site = await TEST_DB_CLIENT.db()
-        .collection(SiteModel.collection.name)
-        .findOne({ name: 'TEST SITE' });
+  it('Should get a single id with :siteId.', async () => {
```

Comment:
Do we need a test for the failures of getting a site?

Reply:
yes